node/src/actors/chain_manager/mod.rs

```diff
@@ -2485,22 +2503,25 @@ fn current_committee_size_requirement(
     last_consolidated_checkpoint: u32,
     current_checkpoint: u32,
     last_checkpoint_signed_by_bootstrap: u32,
+    min_committee_size: u32,
 ) -> u32 {
     assert!(last_consolidated_checkpoint <= current_checkpoint, "Something went wrong as the last consolidated checkpoint is bigger than our current checkpoint {} > {}", last_consolidated_checkpoint, current_checkpoint);
     // If the last consolidated superblock or the current checkpoint is below last_checkpoint_signed_by_bootstrap, return the default committee size
     if last_consolidated_checkpoint <= last_checkpoint_signed_by_bootstrap {
         default_committee_size
     } else if current_checkpoint - last_consolidated_checkpoint >= decreasing_period {
+        // Decrease committee size
+        // The minimum committee size is configurable, but it must be at least 1
```

Comment:
Wouldn't say it's "configurable". It is indeed from a code perspective, but this statement gives the false idea that each user is free to put whatever minimum size they want.